### DISCUSSION TITLE
Struggling with cell identification

### DISCUSSION THREAD

**Discussion by ethanbb** [flatironinstitute/CaImAn#1351] [Q&A]
Hi folks,

I've been trying to refine my CNMF parameters to get fewer false negatives in a sample recording. That is to say, I see lots of blobs in my local correlation image that are obviously (I think??) cells, and while most of them are identified pretty robustly, some have been stubbornly resisting. This is what initial fit looks like currently:

![cnmf_troubleshooting](https://github.com/flatironinstitute/CaImAn/assets/8973825/5d85f186-3a08-40e9-87ba-700bb597d956)

And here are my current params: [cnmf_params.json](https://github.com/flatironinstitute/CaImAn/files/15299310/cnmf_params.json). Here are the relevant ones I've overridden:

```python
init =  {
    'nb': 2,
    'K': 30,
    'gSig': np.array([7, 7]),
    'method_init': 'greedy_roi',
    'ssub': 1,
    'tsub': 1
}
merging = {
    'merge_thr': 0.85
},
patch = {
    'rf': 60,
    'stride': 40
}
```

I recently increased the patch size from 22 to 60 and increased K from 5 to 18, then 30. I noticed that the neurons I was having trouble with were a bit larger than I thought (~30 pixels in diameter), so I thought my previous settings were too small. And also that increasing K would help to recognize more cells; however, it only seems to increase the number of ROIs that are ambiguous/could not be picked out by the naked eye.

Here is the quilt image in case it's helpful (note I am processing 4 planes concatenated along the x axis in 2D):
![quilt_allplanes_2024-05-13](https://github.com/flatironinstitute/CaImAn/assets/8973825/1e7bfe9d-cedd-4325-aa95-758beef984c2)

I would appreciate any suggestions for what to try next!

**Comment by kushalkolar** [comment]
`gSig` is the key parameter for the greedy initialization, I would play with that parameter.

**Reply by ethanbb** [reply]
Thanks, I'll try increasing it more.
